pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/SingleConnectionBrokerRequestHandler.java

```diff
@@ -130,6 +130,7 @@ protected BrokerResponse processBrokerRequest(long requestId, BrokerRequest orig
       _brokerMetrics.addMeteredTableValue(rawTableName, BrokerMeter.BROKER_RESPONSES_WITH_PARTIAL_SERVERS_RESPONDED, 1);
     }
     _brokerMetrics.addMeteredTableValue(rawTableName, BrokerMeter.TOTAL_SERVER_RESPONSE_SIZE, totalResponseSize);
+    _brokerMetrics.addMeteredTableValue(rawTableName, BrokerMeter.RESPONSE_SIZE, totalResponseSize);
```

Comment:
This is exactly the same as `BrokerMeter.TOTAL_SERVER_RESPONSE_SIZE`, do we want to emit 2 meters with the same values?

Reply:
@Jackie-Jiang Have discussed this with @mcvsubbu, we decide to remove this meter and only add a table level requestSize meter.